docs/reference/runtime-fields.asciidoc

```diff
@@ -1,15 +1,17 @@
-[[alerts-runtime-fields]]
-== Add runtime fields to detection alerts
+[[runtime-fields]]
+== Create runtime fields in {elastic-sec}
 
 Runtime fields are fields that you can add to documents after you've ingested your data. For example, you could combine two fields and treat them as one, or perform calculations on existing data and use the result as a separate field. Runtime fields are evaluated when a query is run. 
 
-You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all {elastic-sec} alerts.
+You can create a runtime field and add it to your detection alerts or events from any page that lists alerts or events in a data grid table, such as *Alerts*, *Timelines*, *Hosts*, and *Users*. Once created, the new field is added to the current <<data-views-in-sec,data view>> and becomes available to all {elastic-sec} alerts and events in the data view.
 
-NOTE: Runtime fields can impact performance, because they're evaluated each time a query runs. Refer to {ref}/runtime.html[Runtime fields] for more information.
+NOTE: Runtime fields can impact performance because they're evaluated each time a query runs. Refer to {ref}/runtime.html[Runtime fields] for more information.
 
-To add a runtime field to detection alerts:
+To create a runtime field:
 
-. Go to *Detect* -> *Alerts*, then click the *Fields* toolbar button at the upper-left of the *Alerts* table. The *Fields* browser opens.
+. Go to a page that lists alerts or events (for example, *Detect* -> *Alerts* or *Investigate* -> *Timelines* -> *_Name of Timeline_*).
```

Comment:
This example is using the old navigation hierarchy, **Detect** and **Investigate** groups do not exist anymore, **Alerts** and **Timelines** pages are at the top level now.

Reply:
Thanks @semd, great catch! Updated in a new commit.